airgo/refresh_data.go

```diff
@@ -0,0 +1,98 @@
+package airgo
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	airtable "github.com/mottaquikarim/go-airtable"
+	log "github.com/sirupsen/logrus"
+)
+
+type RefreshOpts struct {
+	ApiKey string
+	BaseId string
+	Conf   string
+	Dest   string
+}
+
+type Config struct {
+	Nav []ConfigItem
+}
+
+type ConfigItem struct {
```

Comment:
lines 24-29 we are defining the variable names in our config Json file and putting them in a struct so we can access them all at once.